tests/api/test_dynamo.py

```diff
@@ -0,0 +1,225 @@
+from hyp3_api import dynamo
+
+
+def test_query_jobs_by_user(tables):
+    table_items = [
+        {
+            'job_id': 'job1',
+            'user_id': 'user1',
+            'job_type': 'fake job type',
+            'status_code': 'fake job stats',
+            'request_time': '2000-10-01T00:00:00+00:00'
+        },
+        {
+            'job_id': 'job2',
+            'user_id': 'user1',
+            'job_type': 'fake job type',
+            'status_code': 'fake job stats',
+            'request_time': '2000-01-01T00:00:00+00:00'
+        },
+        {
+            'job_id': 'job3',
+            'user_id': 'user2',
+            'job_type': 'fake job type',
+            'status_code': 'fake job stats',
+            'request_time': '2000-01-01T00:00:00+00:00'
+        },
+    ]
```

Comment:
Looks like we can define one set of mock records (global or helper function) that can support all of the tests and eliminate having this declaration 6 times

Reply:
probably, but I like that the tests assetions are close to the data declaration, I don't have to scroll to see what I'm putting in the table